implementations/rust/ockam/ockam_abac/src/policy/resources.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{Resource, ResourceName, ResourcesRepository};
+use ockam_core::compat::sync::Arc;
+use ockam_core::Result;
+
+#[derive(Clone)]
+pub struct Resources {
```

Comment:
I mean, that abstraction layer itself does make total sense, but is it useful enough to exist? It's not doing anything that the repository can't do directly

Reply:
I know, I just wanted to keep the consistency we currently have where no repositories are directly accessible from the `CliState`.